FHIR-Converter/package.json

```diff
@@ -225,6 +225,7 @@
 		"watch": "tsc -b -w",
 		"postinstall": "cd client && npm install && sh ./install-engine.sh && cd ../server && npm install && cd ..",
 		"test": "npm run compile && node ./client/out/test/runTest.js && node ./server/out/test/runTest.js",
+		"download-templates": "cd test-data && sh download-templates.sh",
```

Comment:
Is this script only called in build pipeline test?

Reply:
Yes, only for testing.